test/__mocks__/fields/date.mock.ts

```diff
@@ -20,7 +20,13 @@ export const DATE_OF_BIRTH = {
     "filing_date-day": "1",
     "filing_date-month": "1",
     "filing_date-year": "2022",
-  }, DATE = { day: "1", month: "1", year: "2000" }, EMPTY_DATE = { day: "", month: "", year: "" },
+  }, RESIGNED_ON_DATE = {
+    "resigned_on-day": "1",
+    "resigned_on-month": "1",
+    "resigned_on-year": "2023",
+  },
+
+  DATE = { day: "1", month: "1", year: "2000" }, EMPTY_DATE = { day: "", month: "", year: "" },
```

Comment:
not ours but a newline could move from line 28 to 3rd, on 29

Reply:
done!